CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 #### Fixes
 
+ * [#2328](https://github.com/ruby-grape/grape/pull/2328): Don't cache Class.instance_methods - [@byroot](https://github.com/byroot).
```

Comment:
🤦 there's an extra space in front of `*`

SORRY

Reply:
No worries, that's on me for doing these silly mistakes.